src/components/DropCard/DropCard.tsx

```diff
@@ -107,7 +109,8 @@ export const DropCard: FC<DropCardProps> = ({
                 (externalLink ? MintType.External : MintType.ThirdWeb)
               }
               openSeaLink={openSeaLink}
-              dataSuffix={dataSuffix || siteDataSuffix}
+              dataSuffix={dataSuffix}
+              dropDataSuffix={dropDataSuffix}
```

Comment:
open to this: was thinking we'd just use dataSuffix and usually it has no value but in some cases value is set in the Drop config and (in this case) we persist it if box is checked